scvi/_types.py

```diff
@@ -5,7 +5,10 @@
 import mudata
 import torch
 
+from ._compat import Literal
+
 Number = Union[int, float]
 AnnOrMuData = Union[anndata.AnnData, mudata.MuData]
 Tensor = Union[torch.Tensor, jnp.ndarray]
 LossRecord = Union[Dict[str, Tensor], Tensor]
+LatentDataType = Literal["sampled", "dist"]
```

Comment:
I would vote for removing sampled mode asap so we don't introduce a bunch of code we will end up removing anyway